libraries/lib-label-track/LabelTrack.cpp

```diff
@@ -679,11 +702,36 @@ void LabelTrack::Export(wxTextFile & f, LabelFormat format) const
       f.AddLine(wxT("WEBVTT"));
       f.AddLine(wxT(""));
    }
+   else if (format == LabelFormat::PODCAST_CHAPTERS_JSON) {
+      // JSON header
+      // Podcast 2.0 Chapters spec version 1.2.0
+      f.AddLine(wxT("{"));
+      f.AddLine(wxT("  \"version\": \"1.2.0\","));
+      f.AddLine(wxT("  \"chapters\": ["));
+   }
 
    // PRL: to do: export other selection fields
    int index = 0;
-   for (auto &labelStruct: mLabels)
-      labelStruct.Export(f, format, index++);
+   int numLabels = mLabels.size();
+   for (auto &labelStruct: mLabels) {
+      bool isLast = (index == numLabels - 1);
+      labelStruct.Export(f, format, index);
```

Comment:
Please add isLast as an argument to LabelStruct::Export, and move the comma logic there.